doc/content/components/network-server.md

```diff
@@ -7,3 +7,51 @@ weight: 3
 The Network Server handles the LoRaWAN network layer, including MAC commands, regional parameters and adaptive data rate (ADR).
 
 <!--more-->
+
+## Device Management
+
+Network Servers expose `Get`, `Set` and `Delete` gRPC methods for device management. REST analogues for these methods exist as well. Typical clients of these RPC methods are [Console]({{< ref "/components/console.md" >}}) and [CLI]({{< ref "/components/cli.md" >}}).
```

Comment:
I think it's sufficient to say that it exposes the `NsEndDeviceRegistry` service for managing end devices, and refer to `/reference/api/end_device#the-nsenddeviceregistry-service` for details.

I think it's more useful to explain what the NS stores about end devices than the names RPCs it exposes.